### DISCUSSION TITLE
SVG dark mode

### DISCUSSION THREAD

**Discussion by HarryWangATX** [auroral-ui/hexo-theme-aurora#63] [Q&A]
All of my svgs for my blog is built for light mode, is there way to make them work for dark mode?

**Comment by bennyxguo** [comment]
Make sure all your SVG's `stroke` and `fill` property are set to empty string. It will automatically adapt to the theme's colors. For example:

```
stroke=""
fill=""
```

**Reply by HarryWangATX** [reply]
what if I have multiple colors will that still work?
